### ISSUE TITLE
Fix accessibility issues from CI job

### ISSUE THREAD

**Issue by SarahJaine** [HHS/Head-Start-TTADP#159]
In https://github.com/HHS/Head-Start-TTADP/issues/41 we added an accessibility scan to the continuous integration pipeline. This scan is showing us issues that are not "quick fixes"; they'll take some concentrated developer time to understand and fix.

**Acceptance criteria:**
- [ ] fix all accessibility issues highlighted by axe cli tool run in CI pipeline 
- [ ] add `--exit` flag to `axe:ci` script in `/package.json` file, this will make the CI pipeline fail when issues are found

**Accessibility errors as of 11/20**
```
yarn run v1.22.4
$ axe *********************, *********************/activity-reports, *********************/admin --dir ./reports/
Running axe-core 4.0.2 in chrome-headless

Testing ********************* ... please wait, this may take a minute.
  0 violations found!

Testing *********************/activity-reports ... please wait, this may take a minute.
  0 violations found!

Testing *********************/admin ... please wait, this may take a minute.

  Violation of "color-contrast" with 1 occurrences!
    Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds. Correct invalid elements at:
     - #main-content > div
    For details, see: https://dequeuniversity.com/rules/axe/4.0/color-contrast

  Violation of "page-has-heading-one" with 1 occurrences!
    Ensure that the page, or at least one of its frames contains a level-one heading. Correct invalid elements at:
     - html
    For details, see: https://dequeuniversity.com/rules/axe/4.0/page-has-heading-one

2 Accessibility issues detected.
```

**Comment by pamlo412** [comment]
Agreed at grooming that we will timebox work on this issue. If we can fix it, great. If not, it can move lower in priority because the issue only applies to the admin tool, which currently is planned to only have one user, Ryan.

**Comment by rahearn** [comment]
Definition of Done:

- [ ] The business and/or user value of a story has been met
- [ ] CI pipeline green
- [ ] Code is meaningfully tested
- [ ] No linting errors or warnings
- [ ] UI meets WCAG 2.1 - Levels A and AA
- [ ] Code has deployed to a staging environment
- [ ] Security scans have passed
- [ ] [Architectural Decision Records](https://adr.github.io/) are written for major infrastructure decisions with [the Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md).
- [ ] Boundary and Data Flow Diagrams are updated when a change invalidates the current version.
- [ ] Logical Data Model Diagram is updated on DB schema changes.
- [ ] Public API methods are documented when added or changed.
- [ ] OSCAL templates are completed when security controls are implemented or modified.
